src/main/webapp/features/screens/game/dashboard/gameMap/ct-game-map.js

```diff
@@ -42,6 +42,11 @@ angular.module('catan')
                         SelectService.select('edge', edgeId);
                     });
 
+                    element.on('click', DrawMapService.HEX_SELECTOR, function(event) {
```

Comment:
App will try to move robber to the same place and get error. In future, back-end should provide info about places which are available for moving robber here -- like in building of road/settlement/city.